cmd/url.go

```diff
@@ -20,24 +20,27 @@ var urlCmd = &cobra.Command{
 	Use:   "url",
 	Short: "Expose component to the outside world",
 	Long: `Expose component to the outside world.
-The URLs that are generated using this command, can be used to access the
-deployed components from outside the cluster.
-`,
+
+The URLs that are generated using this command, can be used to access the deployed components from outside the cluster.`,
+	Example: fmt.Sprintf("%s\n%s\n%s",
+		urlCreateCmd.Example,
+		urlDeleteCmd.Example,
+		urlListCmd.Example),
 }
 
 var urlCreateCmd = &cobra.Command{
 	Use:   "create [component name]",
 	Short: "Create a URL for a component",
 	Long: `Create a URL for a component.
-The created URL can be used to access the specified component from outside the
-OpenShift cluster.
-`,
-	Example: `# Create a URL for the current component.
-odo url create
 
-# Create a URL for a specific component
-odo url create <component name>
+The created URL can be used to access the specified component from outside the OpenShift cluster.
 `,
+	Example: `  # Create a URL for the current component.
+  odo url create
+
+  # Create a URL for a specific component
+  odo url create myurl
```

Comment:
`myurl` might be a bit misleading. The argument is a name of a component. This looks like it is a name of an url.
`mycomponent` might be better